app/models/booking.rb

```diff
@@ -7,7 +7,12 @@ class Booking < ApplicationRecord
   delegate :code, :type_yard, to: :yard, prefix: true
 
   enum status: {pending: 0, accept: 1, rejected: 2, cancel: 3}
+
   scope :status_asc, ->{order status: :asc}
+  scope :find_date_booking, (lambda do |param, date|
+    where("time_cost_id IN (?) AND booking_date = ?
+    AND status = 1", param, date)
```

Comment:
` AND status = 1` xáo code này ở đây, ở controller sau khi gọi scope  này thì gọi tiếp `.accept` ấy (accept có giá trị là 1 chẳng hạn)